yt_dlp/extractor/nebula.py

```diff
@@ -157,7 +155,7 @@ class NebulaIE(NebulaBaseIE):
     _TESTS = [
         {
             'url': 'https://nebula.app/videos/that-time-disney-remade-beauty-and-the-beast',
-            'md5': 'fe79c4df8b3aa2fea98a93d027465c7e',
+            'md5': '14944cfee8c7beeea106320c47560efc',
```

Comment:
Any idea why the md5s have changed?

Reply:
I don't. But it isn't the first time. I remember that this happened before, that I returned to this extractor/platform after a couple of months and suddenly all MD5s had changed.

Maybe they sometime reencode the videos? Or the metadata is embedded into the MD5, and the metadata schema changes over time?

Either way, they don't change on a daily or weekly basis, but they seem to change every couple of months.